app/controllers/api/v1/research_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class ResearchController < ActionController::API
+
+      def create
+        @research = Research.create(research_params)
+      end
+
+      private
+      def submission_params
```

Comment:
Also, I just realized that the controller filename and declaration need to be the plural of the model so the filename and controller name should be changed to `researches_controller.rb` & `ResearchesController` respectively.